src/common/servlist.c

```diff
@@ -235,6 +235,11 @@ static const struct defaultserver def[] =
 	{"Krstarica", 0},
 	{0,			"irc.krstarica.com"},
 
+#ifdef USE_OPENSSL
+	{"LibertaCasa", 0, 0, 0, LOGIN_SASL, 0, TRUE},
+	{0,			"irc.liberta.casa/+6697"},
```

Comment:
Remove the explicit ports on all of these.

Reply:
OK. I explicitly asked in #hexchat about this and was told to include it.